### PR TITLE
client: add Quorum option in getOption

### LINKED ISSUE #3162: client: cannot set quorum=true
client api has no way to set quorum=true. fleetd currently needs this to avoid getting back a 404 immediately after writing data.

### PR DESCRIPTION
**Pull request by yichengq**
fixes #3162

### CODE REVIEW

**Comment by mischief** [comment]
so far lgtm. someone else should put the rubber stamp though.

File: client/keys.go [hunk 1]
```diff
@@ -194,6 +194,10 @@ type GetOptions struct {
 	// not be sorted and the ordering used should not be considered
 	// predictable.
 	Sort bool
+
+	// Quorum specifies whether it gets the latest committed value that
+	// has been applied in a quorum of peers, which ensures strong consistency.
```
Comment (xiang90) [inline client/keys.go]: quorum of members, which ensures external consistency (or linearizability)

Comment (yichengq) [inline client/keys.go]: changed.

**Comment by xiang90** [comment]
LGTM

**Pull request merged**